File: app/src/main/java/com/eure/traveling/entity/Shot.java
package com.eure.traveling.entity;

import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
import com.fasterxml.jackson.annotation.JsonProperty;

@JsonIgnoreProperties(ignoreUnknown = true)
public class Shot {

    private int id;
    private String title;
    @JsonProperty("likes_count")
    private int likesCount;
    @JsonProperty("images")
    private Image image;
    @JsonProperty("user")
    private Designer designer;

    public Shot() {

    }

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public String getTitle() {
        return title;
    }

    public void setTitle(String title) {
        this.title = title;
    }

    public int getLikesCount() {
        return likesCount;
    }

    public void setLikesCount(int likesCount) {
        this.likesCount = likesCount;
    }

    public Image getImage() {
        return image;
    }

    public void setImage(Image image) {
        this.image = image;
    }

    public Designer getDesigner() {
        return designer;
    }

    public void setDesigner(Designer designer) {
        this.designer = designer;
    }
}

File: app/src/main/java/com/eure/traveling/entity/Shot.kt
package com.eure.traveling.entity

import com.fasterxml.jackson.annotation.JsonIgnoreProperties
import com.fasterxml.jackson.annotation.JsonProperty

@JsonIgnoreProperties(ignoreUnknown = true)
class Shot {

    var id: Int = 0
    var title: String? = null
    @JsonProperty("likes_count")
    var likesCount: Int = 0
    @JsonProperty("images")
    var image: Image? = null
    @JsonProperty("user")
    var designer: Designer? = null
}
